source/regex.tex

```diff
@@ -264,7 +264,9 @@
 
   // \ref{re.const}, regex constants:
   namespace regex_constants {
-    enum error_type;
+    using syntax_option_type = @\textit{T1}@;
+    using match_flag_type = @\textit{T2}@;
+    using error_type = @\textit{T3}@;
   } // namespace regex_constants
```

Comment:
Please remove the namespace comment. We don't really do this in the Standard, but some people have smuggled this in.